src/StoreApi/Utilities/ReserveStock.php

```diff
@@ -24,6 +38,10 @@ final class ReserveStock {
 	public function get_reserved_stock( \WC_Product $product, $exclude_order_id = 0 ) {
 		global $wpdb;
 
+		if ( $this->db_version < 260 ) {
```

Comment:
This is used in a lot of places, I wonder if extracting to a method would make the condition a bit more readable. Example:

```php
if ( $this->has_valid_db_version() )
```